pubspec.yaml

```diff
@@ -4,7 +4,7 @@ homepage: https://github.com/revanced/revanced-manager
 
 publish_to: 'none'
 
-version: 0.0.56+56
+version: 0.0.57
```

Comment:
How will CI handle this initially? Will it overwrite this version once merged to dev?

Reply:
Yep, check the version bumping test.